src/extensions/wcf/Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater/WCFUpdateStep.cs

```diff
@@ -242,25 +242,42 @@ private void FindPath(IProject project)
             }
             else
             {
-                _path!.Add("main", main);
+                _path.MainFile = main.Single();
                 Logger.LogTrace($"This following file: {main.Single()} needs source code update to replace ServiceHost instance.");
-                _path.Add("config", config);
+                _path.Config = config.Single();
                 Logger.LogTrace($"This following config file: {config.Single()} needs to be updated.");
-                _path.Add("proj", new[] { project.GetFile().FilePath });
+                _path.ProjectFile = project.GetFile().FilePath;
                 Logger.LogTrace($"This following project file: {project.GetFile().FilePath} needs to be updated");
 
+                // only updates the namespace for .cs files does not include clients
+                var usingDirectivesUpdates = new List<string>();
                 foreach (var directive in directives)
                 {
-                    Logger.LogDebug($"This .cs file: {directive} needs using directives updates. Adding the path to collection.");
+                    var root = CSharpSyntaxTree.ParseText(File.ReadAllText(directive)).GetRoot();
+                    if (ContainsIdentifier(root, "ChannelFactory") || ContainsIdentifier(root, "ClientBase"))
+                    {
+                        Logger.LogDebug($"This .cs file: {directive} needs using directives updates. Adding the path to collection.");
+                    }
```

Comment:
I don't think this log message is correct here as it's the same message as the else case.